assets/js/components/Work/Tabs/Administrative/Administrative.jsx

```diff
@@ -197,33 +206,152 @@ const WorkTabsAdministrative = ({ work }) => {
                     label="Project Cycle"
                     defaultValue={projectCycle}
                   />
-                ) : (
-                  <p>{projectCycle}</p>
-                )}
+                ) : projectCycle ? (
+                  <Button
+                    isText
+                    data-testid="facet-link"
```

Comment:
Can we make this `data-testid` a bit more specific to the context of the element?   Maybe something like `data-testid="project-cycle-link"`